mage_ai/server/server.py

```diff
@@ -290,19 +293,28 @@ def start_server(
     manage: bool = False,
     dbt_docs: bool = False,
     instance_type: InstanceType = InstanceType.SERVER_AND_SCHEDULER,
+    project_type: ProjectType = ProjectType.STANDALONE,
+    cluster_type: ClusterType = None,
+    project_uuid: str = None,
 ):
     host = host if host else None
     port = port if port else DATA_PREP_SERVER_PORT
     project = project if project else None
 
     # Set project path in environment variable
     if project:
-        project = project = os.path.abspath(project)
+        project = os.path.abspath(project)
     else:
         project = os.path.join(os.getcwd(), 'default_repo')
 
     if not os.path.exists(project):
-        init_repo(project)
+        init_repo(
+            project,
+            project_type=project_type,
+            cluster_type=cluster_type,
+            project_uuid=project_uuid,
+        )
+    update_project_uuid()
```

Comment:
I think you'll need to move it below `set_repo_path`. Since `get_repo_config` relies on the repo path in the environment variable.